assets/src/edit-story/app/media/local/reducer.js

```diff
@@ -46,7 +46,7 @@ const INITIAL_STATE = {
  * @param {Object} obj.payload The details of the action, specific to the action
  * @return {Object} The new state
  */
-function reducer(state = INITIAL_STATE, { type, payload }) {
+export function reducer(state = INITIAL_STATE, { type, payload }) {
```

Comment:
Why this change? The `reducer` is already the default export. No need to have a named export as well.

The [`eslint-plugin-import/no-named-as-default`](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-named-as-default.md) rule now warns about this, as it might cause unintended effects.

Reply:
Good point, I have #3279 to address this.